src/qt/font.cpp

```diff
@@ -469,9 +469,20 @@ void wxNativeFontInfo::SetPixelSize(const wxSize& size)
 
 void wxNativeFontInfo::SetStyle(wxFontStyle style)
 {
-    m_qtFont.setItalic(style == wxFONTSTYLE_ITALIC);
-//case wxFONTSTYLE_SLANT:
-//case wxFONTSTYLE_NORMAL:
+    switch(style)
+    {
+        case( wxFONTSTYLE_ITALIC ):
+            m_qtFont.setItalic(true);
+            break;
+        case( wxFONTSTYLE_NORMAL ):
+            m_qtFont.setStyle(QFont::StyleNormal);
+            break;
+        case( wxFONTSTYLE_SLANT ):
+            m_qtFont.setStyle(QFont::StyleOblique);
+            break;
+        default:
```

Comment:
This seems unnecessary and will prevent compiler from warning if another `wxFONTSTYLE_XXX` is added later, so it should really be removed.